app/server/utils/ussd/kenya_ussd_state_machine.py

```diff
@@ -124,7 +130,9 @@ def is_token_agent(self, user_input):
 
     # TODO: [Philip] Add community token when available
     def upsell_unregistered_recipient(self, user_input):
-        upsell_message = f"{self.user.first_name} {self.user.last_name} amejaribu kukutumia {self.user} lakini hujasajili. Tuma information yako: jina, nambari ya simu, kitambulisho, eneo, na aina ya biashara yako kwa 0757628885."
+        upsell_message = f"{self.user.first_name} {self.user.last_name} amejaribu kukutumia {self.user} lakini " \
```

Comment:
we can also make this i18n now. see the part about placeholders: https://pypi.org/project/python-i18n/